vanilla/scss/styles.scss

```diff
@@ -70,3 +70,28 @@ li > p {
 .admonition-security-notice {
   padding-left: 1rem;
 }
+
+// Toc custom styles
+.p-table-of-contents__nav > ul > li:first-child > a {
+  display: none;
```

Comment:
This feels wrong. If sphinx renders it, why do we hide it? Are we sure it's always something we want to hide?

I would be better to find some configuration option to tell sphinx not to render it.

Reply:
I did do some reading, it seems other people had the same problem. There are a few solutions [here](https://stackoverflow.com/questions/7278253/using-sphinx-how-can-i-remove-the-title-appearing-in-the-side-bars-table-of-co). One of them is to use 

`.. raw:: html` with the title and it will be excluded from the `toc`. The issue with this I think is Daniele didn't want to add anything the users of the docs would need to _know_ to do, like add this specific HTML heading instead of the usual Sphinx one

Reply:
OK. Seems even the Furo theme does it this way with display: none, so we can keep it.

https://github.com/pradyunsg/furo/blob/d0f880da9453d6a5c2363b9b5c24927ed0c2747f/src/furo/assets/styles/components/_table_of_contents.sass#L33